cmd/kops/replace.go

```diff
@@ -97,9 +98,18 @@ func RunReplace(f *util.Factory, cmd *cobra.Command, out io.Writer, c *replaceOp
 	codec := codecs.UniversalDecoder(kopsapi.SchemeGroupVersion)
 
 	for _, f := range c.Filenames {
-		contents, err := vfs.Context.ReadFile(f)
-		if err != nil {
-			return fmt.Errorf("error reading file %q: %v", f, err)
+		var contents []byte
+		if f == "-" {
```

Comment:
`if f == "-"` then shouldn't we be outside of the loop?

Reply:
@chrislovecnm Looking at how kubectl handles it, it's possible to use the command `kubectl create -f -,service.yaml` and `kubectl create -f pod.yaml,-` so it'd make sense for kops to be able to do the same thing. That means that `if f == "-"` needs to be in the loop